fragadelphia_rules_and_regulations.md

```diff
@@ -69,34 +71,51 @@
+&nbsp;1.1.3 Team tickets are non-transferable and non-refundable. Purchaser of team ticket MAY NOT resell or request refund of a Fragadelphia Team Ticket.
+
+&nbsp;1.1.4 Upon ticket purchase completion, ownership of the ticket is assigned to the designated team captain who must be a core player.
+
+### 1.2 Roster Submission Guidelines
+
+&nbsp;1.2.1 During ticket purchase, purchasers will be prompted to designate the team captain. The team captain must be a player from the team's Frag core.
+
+&nbsp;&nbsp;1.2.1.1 Team captains have 1 week after ticket purchase to submit a Frag core of 3-5 players. After this 1 week period, these players are locked as the Frag core. Failure to submit a Frag core within the 1 week period will result in penalty.
+
+&nbsp;&nbsp;1.2.1.2 Designated captains can request the captain role to be transferred to any Frag core member at any time.
 
-&nbsp;1.1.4 If a team is unable to attend but does not inform Fragadelphia through the discord ticketing system 17 days prior to an event, they will be seeded into the tournament and all matches will be recorded as a forfeit.
+&nbsp;&nbsp;1.2.1.3 Teams are required to play all matches with a minimum of 3 or a majority of their submitted Frag core.
 
-### 1.2 Roster Limitations
+&nbsp;1.2.2 Frag rosters will lock 17 days before the tournament. Changes to the team's submitted Frag roster cannot be made after this 17 day period.
 
-&nbsp;1.2.1 All captains must submit team forms by the designated submission date. Failure to do so may result in a penalty.
+&nbsp;&nbsp;1.2.2.1 If a team cannot attend they must inform Fragadelphia using the Discord ticket system before the 17 day roster lock. If a team fails to inform Fragadelphia of their inability to attend after the 17 day roster lock, the submitted Frag roster will be seeded into the tournament and all matches will be recorded as forfeits.
 
-&nbsp;1.2.2 There are no limitations on gender or region for rostered players. All are welcome at Fragadelphia.
+&nbsp;1.2.3 If participation majority as stated in 1.2.1.3 is not affected, teams can submit roster additions at check in of the event. These additions will incur a fee of 1/5th of a full price team pass per player.
 
-### 1.3 Substitutions
+### 1.3 Roster Limitations
 
-&nbsp;1.3.1 A team may make a substitution in-between matches for any reason.
+&nbsp;1.3.2 There are no limitations on gender or region for rostered players. All are welcome at Fragadelphia.
 
-&nbsp;1.3.2 A team may make a substitution during a match or in-between maps only in the case of a medical emergency. 
 
-&nbsp;&nbsp;1.3.2.1 Teams are allowed 1 emergency substitution per event.
+### 1.4 Substitutions
+
+&nbsp;1.3.1 A team may make a substitution in-between matches for any reason, but not to the team's Frag core.
```

Comment:
my only note is the wording here.
"A team may make a substitution in-between matches for any reason, but not to the team's Frag core." is correct but sounds funky. Maybe something more along the lines of "A team may make a substitution in-between matches for any reason, but they must maintain a majority of their Frag core."